spec/style-rules.md

```diff
@@ -37,6 +37,10 @@ To execute a style rule `rule`:
 
 * Execute each child `child` of `rule`.
 
+* If `css` contains any children and `selector` is [bogus], throw an error.
```

Comment:
Should this be updated now even though the current state is just a deprecation warning?

Reply:
Yeah—as far as the spec is concerned, deprecation warnings and intermediate states are essentially an implementation detail.